packages/app-bridge/src/react/useAfterInsertion.tsx

```diff
@@ -0,0 +1,16 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useEffect } from 'react';
+
+import { type AppBridgeBlock } from '../AppBridgeBlock';
+
+export const useAfterInsertion = <T extends AppBridgeBlock>(appBridge: T, callback: () => void, enabled = true) => {
+    useEffect(() => {
+        const isNew = appBridge.context('isNew').get();
+        if (enabled && isNew) {
+            callback();
+        }
+    }, [enabled, callback, appBridge]);
```

Comment:
```suggestion
    }, [enabled, appBridge]);
```
I think we should remove the `callback` here, what do you think?

Reply:
Initially wanted to keep the hook consistent with the rules of React, but since useEffectEvent is still in experimental phase let's go ahead with the removal as it shouldn't harm anything